.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,62 @@
+# Copyright 2022 The Engula Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Bug report
+description: Problems and issues with the software
+labels: [ "bug" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you very much for submitting feedback to Engula to help Engula develop better.
+
+        If it is an idea or help wanted, please go to [Github Discussion](https://github.com/engula/engula/discussions) forum.
```

```suggestion
        If it is an idea or help wanted, please go to [Github Discussion](https://github.com/engula/engula/discussions).
```

"Discussion" and "forum" seem redundant.